android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -1093,7 +1093,7 @@ public void onPackageInstalled(List<Map<String, String>> keyboardsInstalled) {
         hashMap.get(KMManager.KMKey_LanguageID),
         hashMap.get(KMManager.KMKey_LanguageName),
         hashMap.get(KMManager.KMKey_Version),
-        hashMap.get(KMManager.KMKey_HelpLink),
+        hashMap.get(KMManager.KMKey_CustomHelpLink),
```

Comment:
This is a separate fix for displaying offline help. There used to be a distinction between 
KMKey_HelpLink - link to the keyboard page on help.keyman.com/keyboard/ 
KMKey_CustomHelpLink - offline welcome.htm

But now KMKey_HelpLink isn't really used (Keyboard class will generate the help link if welcome.htm doesn't exist)